### PR TITLE
feat(manage-variable): handle duplicate invalid status in variable name

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

- Apply invalid text to variable name form.
- Handle invalid check in edit case.

https://user-images.githubusercontent.com/83635051/210323534-31c51812-a158-4827-8477-380828796e49.mov

### CODE REVIEW

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue [hunk 1]
```diff
@@ -102,6 +103,10 @@ const state = reactive({
         EDIT: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.SUB_TITLE_EDIT'),
     })),
     selectedVariable: '' as string,
+    variableNams: computed<string[]>(() => {
```
Comment (WANZARGEN) [inline src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue:106]: suggestion: fix typo please :)

```suggestion
    variableNames: computed<string[]>(() => {
```

**Review by WANZARGEN — APPROVED** [review]
hat a nice clean code :)
There is a typo. Except for it, I love it!

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue (hunk 1 above)
Comment (piggggggggy) [reply src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue]: OMG... Thanks!!

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableForm.vue [hunk 2]
```diff
@@ -107,10 +114,16 @@ const {
...
+    invalidTexts,
 } = useFormValidator({
     name: '',
 }, {
-    name(value: string) { return value.trim().length > 0; },
+    name(value: string) {
+        if (props.variableNames.includes(value)) {
+            return i18n.t('Name is duplicated');
+        }
+        return value.trim().length > 0;
```
Comment (yuda110) [inline src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableForm.vue:125]: nitpick: Invalid text seems to be needed here too.

Comment (piggggggggy) [reply src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableForm.vue:126]: I add annotation! Thank you!

**Pull request merged**